File: Beecrowd-Problems/C#/done!/problem1005.cs
using System;
using System.Globalization;

class URI
{

    static void Main(string[] args)
    {
        CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
        CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;
        double weightA = 3.5d, weightB = 7.5d;
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double average = (weightA * a + weightB * b) / (weightA + weightB);
        Console.WriteLine($"MEDIA = {average:F5}");
    }
}

File: Beecrowd-Problems/Rust/done!/problem1005.rs
use std::io;

fn read_user_input() -> f64 {
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line.");
    let num: f64 = match input.trim().parse() {
        Ok(num) => num,
        Err(_) => {
            println!("Input must be a floating point value.");
            return read_user_input();
        }
    };
    num
}

fn main() {
    const WEIGHT_A: f64 = 3.5;
    const WEIGHT_B: f64 = 7.5;
    let a: f64 = read_user_input();
    let b: f64 = read_user_input();
    let media: f64 = ((WEIGHT_A * a) + (WEIGHT_B * b)) / (WEIGHT_A + WEIGHT_B);
    println!("MEDIA = {:.5}", media);
}